.cspell

```diff
@@ -84,6 +84,7 @@ lucene
 Lucene
 lycheeverse
 marvinpinto
+memlock
```

Comment:
Are those used anywhere?

Reply:
they were on my local docker-compose.yaml, got it!